EarlGrey/Action/GREYActions.m

```diff
@@ -161,18 +165,7 @@ @implementation GREYActions
     NSString *textStr;
     // If we're dealing with a text field in a web view, we need to use JS to get the text value.
     if ([element isKindOfClass:webElement]) {
-      // Input tags can be identified by having the 'title' attribute set, or current value.
-      // Associating a <label> tag to the input tag does NOT result in an iOS accessibility element.
-      NSString *xPathResultType = @"XPathResult.FIRST_ORDERED_NODE_TYPE";
-      NSString *xPathForTitle =
-          [NSString stringWithFormat:@"//input[@title=\"%@\" or @value=\"%@\"]",
-              [element accessibilityLabel], [element accessibilityLabel]];
-      NSString *jsForTitle = [[NSString alloc] initWithFormat:
-          @"document.evaluate('%@', document, null, %@, null).singleNodeValue.value = '';",
-          xPathForTitle,
-          xPathResultType];
-      UIWebView *parentWebView = (UIWebView *)[element grey_viewContainingSelf];
-      textStr = [parentWebView stringByEvaluatingJavaScriptFromString:jsForTitle];
+      textStr = [GREYActions grey_webviewInputValue:element];
```

Comment:
Looks like grey_webviewInputValue: not only gets the text but also sets it to "" so there is no point falling through this if (and injecting backspaces)